src/Render/FriendicaSmarty.php

```diff
@@ -47,11 +48,16 @@ function __construct(string $theme, array $theme_info)
 		$template_dirs = $template_dirs + ['base' => 'view/' . self::SMARTY3_TEMPLATE_FOLDER . '/'];
 		$this->setTemplateDir($template_dirs);
 
-		$this->setCompileDir('view/smarty3/compiled/');
-		$this->setConfigDir('view/smarty3/');
-		$this->setCacheDir('view/smarty3/');
 
-		$this->left_delimiter = Renderer::getTemplateLeftDelimiter('smarty3');
+		$compile_dir = DI::config()->get('smarty3', 'compile_dir') ?? DI::basePath() . '/view/smarty3/compiled/';
+		$config_dir  = DI::config()->get('smarty3', 'config_dir')  ?? DI::basePath() . '/view/smarty3/';
+		$cache_dir   = DI::config()->get('smarty3', 'cache_dir')   ?? DI::basePath() . '/view/smarty3/';
```

Comment:
`Config->get` has a default value parameter we can use here.
```suggestion
		$compile_dir = DI::config()->get('smarty3', 'compile_dir', DI::basePath() . '/view/smarty3/compiled/');
		$config_dir  = DI::config()->get('smarty3', 'config_dir', DI::basePath() . '/view/smarty3/');
		$cache_dir   = DI::config()->get('smarty3', 'cache_dir', DI::basePath() . '/view/smarty3/');
```